src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -856,7 +864,8 @@ public String getBoxAuthzTokens(HttpServletRequest request) {
 			rv = BoxUtils.getBoxAccessToken(userId);
 		}
 
-		return rv != null ? "<link rel=\"stylesheet\" href=\"vendors/bootstrap/bootstrap.min.css\"><div class=\"jumbotron\"><h1 role=\"alert\">Authorized!</h1><p>You can now select a Box folder and migrate project sites to it.</p><p><a target=\"top\" href=\"/\">Go back</a></p></div>" : "<link rel=\"stylesheet\" href=\"vendors/bootstrap/bootstrap.min.css\"><div class=\"jumbotron\"><h1 role=\"alert\">Unauthorized!</h1><p>You need to authorize Box.<a target=\"_top\" href=\"/\">Go back</a></p></div>";
+		return rv != null ? "<link rel=\"stylesheet\" href=\"vendors/bootstrap/bootstrap.min.css\"><div class=\"jumbotron\"><h1 role=\"alert\">Authorized!</h1><p>You can now select a Box folder and migrate project sites to it.</p><p><a target=\"top\" href=\"/\">Go back</a></p></div>"
+				: "<link rel=\"stylesheet\" href=\"vendors/bootstrap/bootstrap.min.css\"><div class=\"jumbotron\"><h1 role=\"alert\">Unauthorized!</h1><p>You need to authorize Box.<a target=\"_top\" href=\"/\">Go back</a></p></div>";
```

Comment:
This return looks long for readability. Take this out in to a simple function instead. Also i see that text inside the condition is presented to the User this should be coming from a properties file than inlining it here